### PR TITLE
Try to fix the logtostderr issue

### PR DESCRIPTION
**Pull request by pierluigilenoci**
Ref: #37 and #44

### CODE REVIEW

**Comment by pierluigilenoci** [comment]
@maciaszczykm @floreks I tried to make a change that did not break previous installations.
With this option, it is now possible to configure where to send the logs and a meaningless error message disappears.

Let me know what you think about it.

File: README.md [hunk 1]
```diff
@@ -9,6 +9,9 @@ Small binary to scrape and store a small window of metrics from the Metrics Serv
 | db-file  | What file to use as a SQLite3 database.  |  `/tmp/metrics.db` |
 | metric-resolution | The resolution at which dashboard-metrics-scraper will poll metrics.  | `1m` |
 | metric-duration | The duration after which metrics are purged from the database. | `15m` |
+| log-level | The log level. | `info` |
+| logtostderr | Log to stderr. | `true` |
+| namespace | The namespace to use for all metric calls. When provided, skip node metrics. | `defaults to cluster level metrics` |
```
Comment (maciaszczykm) [inline README.md]: I'd remove the quotation marks in the defaults column.

File: README.md [hunk 2]
```diff
@@ -9,6 +9,9 @@ Small binary to scrape and store a small window of metrics from the Metrics Serv
 | db-file  | What file to use as a SQLite3 database.  |  `/tmp/metrics.db` |
 | metric-resolution | The resolution at which dashboard-metrics-scraper will poll metrics.  | `1m` |
 | metric-duration | The duration after which metrics are purged from the database. | `15m` |
+| log-level | The log level. | `info` |
+| logtostderr | Log to stderr. | `true` |
```
Comment (maciaszczykm) [inline README.md]: s/stderr/standard error

**Comment by maciaszczykm** [comment]
Looks good to me. @floreks do you have anything to add?

File: README.md (hunk 1 above)
Comment (pierluigilenoci) [reply README.md:14]: Do you mean across the column? Or just the ones I added?

File: README.md (hunk 2 above)
Comment (pierluigilenoci) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (maciaszczykm) [reply README.md]: It's good now.

**Comment by pierluigilenoci** [comment]
@floreks could you please take a look?

**Comment by floreks** [comment]
@pierluigilenoci tests are failing

**Comment by pierluigilenoci** [comment]
@floreks could you please add `/ok-to-test` command?

**Comment by floreks** [comment]
Linter still fails

**Comment by pierluigilenoci** [comment]
Thank you @floreks and @maciaszczykm ❤️

**Pull request merged**